charts/kamaji-etcd/templates/etcd_sts.yaml

```diff
@@ -24,7 +24,6 @@ spec:
         {{- toYaml .Values.podAnnotations | nindent 8 }}
         {{- end }}      
     spec:
-      serviceAccountName: {{ include "etcd.serviceAccountName" . }}
```

Comment:
The etcd cluster didn't ever need a SA since there's no interaction with the Kubernetes API Server.